app/src/main/scala/org/alephium/explorer/persistence/schema/OutputSchema.scala

```diff
@@ -90,5 +90,11 @@ object OutputSchema extends SchemaMainChain[OutputEntity]("outputs") {
   def createNonSpentIndex(): DBActionW[Int] =
     sqlu"create unique index if not exists non_spent_output_idx on #${name} (address, main_chain, key, block_hash) where spent_finalized IS NULL;"
 
+  def createNonSpentOutputCoveringIndex(): DBActionW[Int] =
```

Comment:
Do we need to use the INCLUDE keyword to create this covering index? Would be great to run a `EXPLAIN ANALYZE` for the balance query to check if we are doing index only scan.

Reply:
as discussed privately. Using `INCLUDE` provide same performance, but reduce the index size: 
```
explorer=# SELECT 
    pg_size_pretty(pg_relation_size('non_spent_output_covering_idx')) AS index_size;
 index_size 
------------
 1085 MB
(1 row)

explorer=# SELECT 
    pg_size_pretty(pg_relation_size('non_spent_output_covering_include_idx')) AS index_size;
 index_size 
------------
 690 MB
(1 row)
```

So I just pushed a commit to use that.